<launch>
	<!-- ========================================================== -->
	<!-- Configuration of RECORD ROT.                               -->
    <!--                                                            -->
    <!-- Launch this file to run the experiment, as in:             -->
    <!-- roslaunch experiments recordrot.launch                     -->
    <!--                                                            -->
	<!-- ========================================================== -->

	<!-- ***  Core stuff  *** -->
    <include file="$(find flycore)/launch/main.launch" />



	<!-- ***  The Experiment  *** -->
    <param name="save/imagetopic" type="string" value="camera/image_rect"/>
	<node pkg="experiments" type="RecordRot.py" name="Experiment" required="true" />
  
</launch>
